utils/appliance/implementations/ui.py

```diff
@@ -352,12 +353,20 @@ def pre_navigate(self, *args, **kwargs):
     def post_navigate(self, *args, **kwargs):
         pass
 
-    def log_message(self, msg):
-        logger.info("[UI-NAV/{}/{}]: {}".format(self.obj.__class__.__name__, self._name, msg))
+    def log_message(self, msg, level="debug"):
```

Comment:
we should really already use a logging adapter instead of creating a logging-ish adaption api